docs/content/docs/guide/metrics.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Running the PipelineRun
+weight: 4
+---
+
+# Metrics Overview
+
+The metrics for pipelines-as-code can be accessed through the `pipelines-as-code-watcher` service on port `9090`.
+
+pipelines-as-code supports various exporters, such as Prometheus, Google Stackdriver, and more.
```

Comment:
maybe add links to it

Reply:
I can add generic Prometheus and Google Stackdriver for Prometheus is that you mean?

Reply:
yeah